[2025-09-11T08:00:03.054+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-11T08:00:03.093+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T07:00:00+00:00 [queued]>
[2025-09-11T08:00:03.104+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T07:00:00+00:00 [queued]>
[2025-09-11T08:00:03.105+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-11T08:00:03.120+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): make_prediction> on 2025-09-11 07:00:00+00:00
[2025-09-11T08:00:03.130+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_hourly_predictions', 'make_prediction', 'scheduled__2025-09-11T07:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/tmp/tmptk066rtq']
[2025-09-11T08:00:03.134+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask make_prediction
[2025-09-11T08:00:03.134+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1112) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-11T08:00:03.135+0000] {standard_task_runner.py:64} INFO - Started process 1114 to run task
[2025-09-11T08:00:03.215+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_hourly_predictions.make_prediction scheduled__2025-09-11T07:00:00+00:00 [running]> on host 02ae08abed04
[2025-09-11T08:00:03.526+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_hourly_predictions' AIRFLOW_CTX_TASK_ID='make_prediction' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-11T07:00:00+00:00'
[2025-09-11T08:00:03.528+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-11T08:00:03.551+0000] {logging_mixin.py:188} INFO - üîÆ Making delay prediction...
[2025-09-11T08:00:52.777+0000] {logging_mixin.py:188} INFO - ‚ùå Not enough recent data
[2025-09-11T08:00:52.981+0000] {python.py:237} INFO - Done. Returned value was: insufficient_data
[2025-09-11T08:00:52.986+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-11T08:00:53.436+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_hourly_predictions, task_id=make_prediction, run_id=scheduled__2025-09-11T07:00:00+00:00, execution_date=20250911T070000, start_date=20250911T080003, end_date=20250911T080053
[2025-09-11T08:00:53.660+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-11T08:00:53.826+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-11T08:00:53.831+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
